encodings/README.md

```diff
@@ -0,0 +1,15 @@
+# OpenCensus Library Encoding Package
+This documentation serves to document the on-the-wire encoding format supported in
+OpenCensus. It describes the key types and the overall behavior.
+
+## Formats
+* [Binary Encoding](BinaryEncoding.md)
```

Comment:
btw, we are working towards making it a standard https://github.com/w3c/trace-context-binary/pull/2 I'd appreciate your review on that PR